pkg/jobs/jobsprofiler/profilerconstants/constants.go

```diff
@@ -20,6 +20,7 @@ import (
 )
 
 const DSPDiagramInfoKeyPrefix = "~dsp-diag-url-"
+const DSPDiagramInfoKeyMax = DSPDiagramInfoKeyPrefix + ":"
```

Comment:
Could see adding a small comment reminding folks that `:` is the ASCII character after 9.

Reply:
Done.